modules/apps/portal-search/portal-search-test-util/src/main/java/com/liferay/portal/search/test/util/sort/BaseNestedFieldsSortTestCase.java

```diff
@@ -59,12 +58,10 @@ protected void addDocumentWithOneDDMField(
 	}
 
 	protected void assertSort(String fieldName) {
-		Stream.of(
-			"C", "B", "A"
-		).forEach(
-			value -> addDocumentWithOneDDMField(
```

Comment:
@julschong Please double check if there's still usage of `addDocumentWithOneDDMField`.